pkg/installer/verify

```diff
@@ -9,7 +9,7 @@
 
 #storage device benchmarking
 mkdir "$REPORT/storage-performance/"
-for i in $(lsblk -anlb -o "TYPE,NAME,SIZE" | grep "^disk" | awk '$3 { print $2;}'); do
+for i in $(lsblk -anlb -o "TYPE,NAME,SIZE" | grep "^disk" | awk '$3 { print $2;}' | grep -v "^ram.*" 2> /dev/null); do
```

Comment:
I'm just curious: why is stderr redirection necessary here?

Reply:
not needed, I thought in remove it to not mess up the for loop output (from fio), but you are right, it's better to keep printing errors... PR was updated...